x-pack/plugins/index_management/public/application/components/template_form/steps/step_logistics.tsx

```diff
@@ -24,70 +32,125 @@ import { schemas, nameConfig, nameConfigWithoutValidations } from '../template_f
-      defaultMessage: 'A unique identifier for this template.',
-    }),
-    testSubject: 'nameField',
-  },
-  indexPatterns: {
-    title: i18n.translate('xpack.idxMgmt.templateForm.stepLogistics.indexPatternsTitle', {
-      defaultMessage: 'Index patterns',
-    }),
-    description: i18n.translate(
-      'xpack.idxMgmt.templateForm.stepLogistics.indexPatternsDescription',
-      {
-        defaultMessage: 'The index patterns to apply to the template.',
-      }
-    ),
-    testSubject: 'indexPatternsField',
-  },
-  order: {
-    title: i18n.translate('xpack.idxMgmt.templateForm.stepLogistics.orderTitle', {
-      defaultMessage: 'Merge order',
-    }),
-    description: i18n.translate('xpack.idxMgmt.templateForm.stepLogistics.orderDescription', {
-      defaultMessage: 'The merge order when multiple templates match an index.',
-    }),
-    testSubject: 'orderField',
-  },
-  priority: {
-    title: i18n.translate('xpack.idxMgmt.templateForm.stepLogistics.priorityTitle', {
-      defaultMessage: 'Merge priority',
-    }),
-    description: i18n.translate('xpack.idxMgmt.templateForm.stepLogistics.priorityDescription', {
-      defaultMessage: 'The merge priority when multiple templates match an index.',
-    }),
-    testSubject: 'priorityField',
-  },
-  version: {
-    title: i18n.translate('xpack.idxMgmt.templateForm.stepLogistics.versionTitle', {
-      defaultMessage: 'Version',
-    }),
-    description: i18n.translate('xpack.idxMgmt.templateForm.stepLogistics.versionDescription', {
-      defaultMessage: 'A number that identifies the template to external management systems.',
-    }),
-    testSubject: 'versionField',
-  },
-};
+function getFieldsMeta(esDocsBase: string) {
+  return {
+    name: {
+      title: i18n.translate('xpack.idxMgmt.templateForm.stepLogistics.nameTitle', {
+        defaultMessage: 'Name',
+      }),
+      description: i18n.translate('xpack.idxMgmt.templateForm.stepLogistics.nameDescription', {
+        defaultMessage: 'A unique identifier for this template.',
+      }),
+      testSubject: 'nameField',
+    },
+    indexPatterns: {
+      title: i18n.translate('xpack.idxMgmt.templateForm.stepLogistics.indexPatternsTitle', {
+        defaultMessage: 'Index patterns',
+      }),
+      description: i18n.translate(
+        'xpack.idxMgmt.templateForm.stepLogistics.indexPatternsDescription',
+        {
+          defaultMessage: 'The index patterns to apply to the template.',
+        }
+      ),
+      testSubject: 'indexPatternsField',
+    },
+    dataStream: {
+      title: i18n.translate('xpack.idxMgmt.templateForm.stepLogistics.dataStreamTitle', {
+        defaultMessage: 'Data stream',
+      }),
+      description: (
+        <FormattedMessage
+          id="xpack.idxMgmt.templateForm.stepLogistics.dataStreamDescription"
+          defaultMessage="Wheter indices that match the index patterns should automatically create a data stream. {docsLink}"
```

Comment:
"Wheter" is misspelled, and I don't think we need a line break between these two sentences. I'd also suggest rewording the text as an action but would like to hear @jrodewig's input:

> Create a data stream instead of an index.